src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyApprovalServiceTests.cs

```diff
@@ -139,6 +138,17 @@ await using (var context = InMemoryApplicationDbContext(contentDbContextId))
                 Assert.Equal(Immediately, updatedMethodology.PublishingStrategy);
                 Assert.True(updatedMethodology.Updated.HasValue);
                 Assert.InRange(DateTime.UtcNow.Subtract(updatedMethodology.Updated!.Value).Milliseconds, 0, 1500);
+
+                var statusList = await context
+                    .MethodologyStatus
+                    .Where(ms => ms.MethodologyVersionId == methodologyVersion.Id)
+                    .ToListAsync();
+                var status = Assert.Single(statusList);
+                Assert.Equal(methodologyVersion.Id, status.MethodologyVersionId);
+                Assert.Equal("Test approval", status.InternalReleaseNote);
+                Assert.Equal(Approved, status.ApprovalStatus);
+                Assert.InRange(DateTime.UtcNow.Subtract(status.Created!.Value).Milliseconds, 0, 1500);
+                Assert.Equal(UserId, status.CreatedById);
```

Comment:
I would say these lines are not necessary because this is a variant of the unit test specific to methodology images and the status is already covered by `UpdateApprovalStatus_ApprovingMethodologyAddsMethodologyStatus`. This is just duplication of that.